src/deepmol/compound_featurization/rdkit_descriptors.py

```diff
@@ -423,7 +424,10 @@ def __init__(self, mandatory_generation_of_conformers):
         self.mandatory_generation_of_conformers = mandatory_generation_of_conformers
         if self.mandatory_generation_of_conformers:
             self.three_dimensional_generator = ThreeDimensionalMoleculeGenerator()
-        super().__init__()
+
+        if "n_jobs" in kwargs:
```

Comment:
Why don't 3D descriptors support parallelization?

Reply:
because it already uses parallelization internally, and somehow it does not work with joblib

Reply:
but I think this can be further investigated